app/controllers/teachers_controller.rb

```diff
@@ -0,0 +1,18 @@
+class TeachersController < ApplicationController
+    def create
+        @teacher = Teacher.new teacher_params
+        if @teacher.save
+            cookies[:saved_teacher] = true
+            redirect_to root_path
+        else
+            redirect_to root_path, alert: "Failed to submit information :("
```

Comment:
Seems like this alert doesn't show up in the webpage (or in Chrome's javascripit console yet.)

I've manually checked that a validate form shows up in the development database. That's great!

Reply:
Yeah this alert is broken so feel free to fix it if you know how after I merge because we couldn't figure it out.

Reply:
Oh. Sorry I didn't see flash broken in the commit message.